fs_storage/__manifest__.py

```diff
@@ -6,16 +6,17 @@
     "name": "Storage Bakend",
     "summary": "Implement the concept of Storage with amazon S3, sftp...",
     "version": "16.0.1.0.1",
-    "category": "Storage",
+    "category": "FS Storage",
     "website": "https://github.com/OCA/storage",
-    "author": " Akretion, Odoo Community Association (OCA)",
+    "author": " Akretion, ACSONE SA/NV, Odoo Community Association (OCA)",
     "license": "LGPL-3",
-    "development_status": "Production/Stable",
+    "development_status": "Beta",
     "installable": True,
-    "depends": ["base", "component", "server_environment"],
```

Comment:
@florian-dacosta it's an open question... I tend to prefer to have glue addon than having too much dependencies.

Reply:
Well, before, we'd have different fields for each protocol (sftp_login, ftp_login, etc). Including server-env was avoiding a lot of small server-env glue module.
Now, we would just need one, since every thing is in the json field and we don't have extra fields per protocol.
So, no problem for me to have a glue module.